test/index.test.js

```diff
@@ -1222,4 +1229,46 @@ Previous tag: ''
       expect.assertions(1)
     })
   })
+  describe('config error handling', () => {
+    it('schema error', async () => {
+      getConfigMock('config-with-schema-error.yml')
+
+      const payload = require('./fixtures/push')
+
+      await probot.receive({
+        name: 'push',
+        payload
+      })
+      expect(logger).toHaveBeenCalledWith(
+        expect.objectContaining({
+          msg: 'Invalid config file',
+          err: expect.objectContaining({
+            message: expect.stringContaining(
+              '"search" is required and must be a regexp or a string'
+            )
+          })
+        })
+      )
+    })
+    it('yaml exception', async () => {
```

Comment:
Needs a blank line for readability :)

```suggestion
    
    it('yaml exception', async () => {
```